notebooks/tutorial/7 - Getting data from the ADX.ipynb

```diff
@@ -0,0 +1,417 @@
+    "cost_per_gb_transfer_from_adx = 0.00 # for US East (Virginia)\n",
+    "cost_per_gb_transfer_out = 0.09 # for US East (Virginia)\n",
+    "\n",
+    "## Estimate for example EU region\n",
+    "# cost_per_get = 0.00000042 # for EU (Paris)\n",
+    "# cost_per_put = 0.0000053 # for EU (Paris)\n",
+    "# cost_per_gb_per_month = 0.024 # for EU (Paris)\n",
+    "# cost_per_gb_transfer_from_adx = 0.02 # for EU (Paris)\n",
+    "# cost_per_gb_transfer_out = 0.09 # for EU (Paris)\n",
+    "\n",
+    "## Estimate for example Asia Pacific region\n",
+    "# cost_per_get = 0.00000035 # for Asia Pacific (Seoul)\n",
+    "# cost_per_put = 0.0000045 # for Asia Pacific (Seoul)\n",
+    "# cost_per_gb_per_month = 0.025 # for Asia Pacific (Seoul)\n",
+    "# cost_per_gb_transfer_from_adx = 0.08 # for Asia Pacific (Seoul)\n",
+    "# cost_per_gb_transfer_out = 0.11 # for Asia Pacific (Seoul)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "46091bd0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "matches_per_revision = 8717/30 # April average\n",
+    "# matches_per_revision = 9892/28 # Feb average\n",
+    "GB_per_match = 0.03154\n",
+    "files_per_match = 33"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c13e21cb",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "days_per_month = 30\n",
+    "\n",
+    "number_of_matches = number_of_revisions*matches_per_revision\n",
+    "number_of_files = number_of_matches*files_per_match\n",
+    "storage_volume_GB = number_of_matches*GB_per_match\n",
+    "\n",
+    "storage_cost = storage_volume_GB*cost_per_gb_per_month*days_to_store_on_s3/days_per_month\n",
+    "\n",
+    "transfer_put_cost = cost_per_put*number_of_files\n",
+    "transfer_get_cost = cost_per_get*number_of_files\n",
+    "\n",
+    "transfer_volume_cost = storage_volume_GB*cost_per_gb_transfer_from_adx\n",
+    "\n",
+    "download_cost = storage_volume_GB*cost_per_gb_transfer_out\n",
+    "\n",
+    "print(f\"{number_of_revisions} revisions will be {round(storage_volume_GB,2)} GB, \"\n",
+    "     f\"{number_of_matches} matches, and {number_of_files} files.\")\n",
+    "print(f\"Cost to transfer to your S3 bucket: ${round(transfer_put_cost+transfer_volume_cost,3)}\")\n",
+    "print(f\"Cost to store data in S3 for {days_to_store_on_s3} days: ${round(storage_cost,3)}\")\n",
+    "print(f\"Cost to transfer from S3 to local: ${round(transfer_get_cost+download_cost,3)}\")\n",
+    "print(f\"Total cost: ${round(transfer_put_cost+transfer_volume_cost+storage_cost+transfer_get_cost+download_cost,3)}\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1413df7e",
+   "metadata": {},
+   "source": [
+    "## Transferring revisions from Data Exchange to your S3 bucket\n",
+    "\n",
+    "By uncommenting the code below, you agree to pay whatever AWS costs you will incur by running this notebook. \n",
```

Comment:
Instead of using uncommenting which is annoying, put a flag here like

```
i_agree_and_understand_the_costs = false
```

Then each cell can have an early return, like

```
if (!i_agree_and_understand_the_costs):
  return

do_expensive_operation()
```

Reply:
But they run one cell at a time anyway

Reply:
There is always the "run all" button